resources/views/page/donate.blade.php

```diff
@@ -0,0 +1,85 @@
+@extends('layout.default')
+
+@section('title')
+    <title>Donate - {{ config('other.title') }}</title>
+@endsection
+
+@section('meta')
+    <meta name="description" content="Donate">
+@endsection
+
+@section('breadcrumbs')
+    <li class="breadcrumbV2">
+        <a href="{{ route('pages.index') }}" class="breadcrumb__link">
+            Pages
+        </a>
+    </li>
+    <li class="breadcrumb--active">
+        Donate
+    </li>
+@endsection
+
+@section('main')
+    <h2>Donation Items</h2>
+    <div class="gird__container--donation">
+        @foreach ($items as $item)
+            <div class="grid__item--donation">
+                <div class="text-center oper-item card_header">
+                    <h1>{{ $item->name }}</h1>
+                </div>
+                <div class="text-center oper-item card">
+                    <span class="text-center">
+                        <p class="type">Type: {{ $item->type ?? 'N/A' }}</p>
+                        <p class="price">${{ $item->price_usd }}@if ($item->id <= 2)<sup>2</sup>@endif</p>
+                        <p class="adventage">{{ $item->seedbonus ?? 0 }} BON points</p>
+                        <p class="adventage">{{ $item->upload ?? 0 }} Upload</p>
+                        <p class="adventage">{{ $item->invites ?? 0 }} invites</p>
+                    </span>
+                </div>
```

Comment:
Please namespace all the classes you use on the page and put the styles in /resources/sass/pages/_donate.scss

Reply:
Fixed in 0821ea98c94c82bb4543e010daff95db058ce81c